daemon/links_unix.go

```diff
@@ -0,0 +1,74 @@
+// +build !windows
```

Comment:
this is only needed on linux, no other platform (eg solaris) was supported then either.